lib/db/schema/patch-045-046.sql

```diff
@@ -0,0 +1,188 @@
+-- Add emails table and corresponding procedures to
+-- create, get and delete them.
+
+CREATE TABLE IF NOT EXISTS emails (
+  normalizedEmail VARCHAR(255) NOT NULL PRIMARY KEY,
+  email VARCHAR(255) NOT NULL,
+  uid BINARY(16) NOT NULL,
```

Comment:
@rfk What are your thoughts on adding an index to `uid` for faster lookups? This table gets queried by uid and by email.